k8s/install/services/cloudflare-datadog/README.md

```diff
@@ -0,0 +1,25 @@
+# Cloudflare -> Datadog sync in K8s
+
+## Prereqs:
+
+- set `KUBECONFIG` environment variable
+- access to unlocked `ee-infra-private` repo
+- helm/tiller
+
+## Installation
+
+```
+export EE_INFRA_PRIVATE_PATH=/path/to/ee-infra-private
+
```

Comment:
I'd like to avoid references to ee-infra-private in this repo if at all possible. We're likely going to rename it soonish, and it's not helpful for contributors external to our team.